Dockerfile

```diff
@@ -1,7 +1,19 @@
+FROM maven:3.8.1-openjdk-11 AS build
```

Comment:
I would rather use GitHub Actions to build the JAR file instead of a build container. Please see https://github.com/samply/store-fhir-adapter/blob/main/.github/workflows/ci.yml

Reply:
I am not familiar with Actions, so it could be some time before I can make the requested change.

Would it be possible to incorporate the build container for the time being, and upgrade to GitHub Actions at a later date?